cmd/routedns/config.go

```diff
@@ -109,8 +109,9 @@ type group struct {
 	EDNS0Data  []byte                  `toml:"edns0-data"`  // EDNS0 modifier option data
 
 	// Failover/Failback options
-	ResetAfter    int  `toml:"reset-after"`    // Time in seconds after which to reset resolvers in fail-back and random groups, default 60.
+	ResetAfter    int  `toml:"reset-after"`    // Time in seconds after which to reset resolvers in fail-back and random groups, default 0 (reset immediately).
```

Comment:
This is changing the default. In Go, a value that isn't being assigned has a default value. In the case of an `int` that would be 0. That's why https://github.com/folbricht/routedns/blob/master/failback.go#L64-L66 is setting the default when the value hasn't been configured in a config. You can switch to using -1 though.

Reply:
If you insist on that then alright I guess. Now it accepts any negative integer. The text is wrong now though because somebody might still want to use 0 because they read "time in seconds" and think 0 will mean 0 seconds.

Reply:
True, that could be clarified in the doc. Basically "not set" means 0, which will turn into 60s as a default. I just want to be very careful about breaking existing configs that do not set this value. It'd change the behavior quite a bit if 0 was actually used.

Reply:
Yeah well I would've preferred to set it to 60 only if it's undefined/not set. But no idea how to code that up in Go. If you could write the code for that that'd be great. Because now I have to tell the user to pick a positive number, but some users may think that this includes 0, so it's getting pretty convoluted.